acceptance/localcluster/localcluster.go

```diff
@@ -38,11 +38,11 @@ const (
 	domain         = "local"
 )
 
-var cockroachImage = flag.String("i", builderImage, "the docker image to run")
-var cockroachBinary = flag.String("b", defaultBinary(), "the binary to run (if image == "+builderImage+")")
-var cockroachEntry = flag.String("e", "", "the entry point for the image")
-var waitOnStop = flag.Bool("w", false, "wait for the user to interrupt before tearing down the cluster")
-var logDirectory = flag.String("l", "", "the directory to store log files, relative to where the test source")
+var cockroachImage = pflag.String("i", builderImage, "the docker image to run")
+var cockroachBinary = pflag.String("b", defaultBinary(), "the binary to run (if image == "+builderImage+")")
+var cockroachEntry = pflag.String("e", "", "the entry point for the image")
+var waitOnStop = pflag.Bool("w", false, "wait for the user to interrupt before tearing down the cluster")
+var logDirectory = pflag.String("l", "", "the directory to store log files, relative to where the test source")
```

Comment:
These flags won't be usable from the "go test" command line. That seems like a serious drawback. Ditto for the logging flags.